tests/test_export.py

```diff
@@ -103,6 +103,8 @@ def setUpClass(cls):
 
         assert Feed.objects.count() == 3
         assert Post.objects.count() == 3
+        cls.root_dir = "/tmp"
+        cls.corpus_dir = "/tmp/corpus"
```

Comment:
I know this hard code is inherited, but while I'm reviewing I'd suggest that we use [`mkdtemp()`](https://docs.python.org/3.5/library/tempfile.html#tempfile.mkdtemp) for tests and ensure that we also clean up the tests by deleting the directory when create when we're done testing.